pullrequest.rst

```diff
@@ -346,7 +346,7 @@ Now you want to
 <https://help.github.com/articles/creating-a-pull-request-from-a-fork/>`_.
 If this is a pull request in response to a pre-existing issue on the
 `issue tracker`_, please make sure to reference the issue number using
-``gh-NNNN`` in the pull request title or message.
+``gh-NNNNN: `` in the pull request title and ``#NNNNN`` in the description.
```

Comment:
It seems that a space inside a code block escapes the following double-backtick:

![image](https://user-images.githubusercontent.com/4881073/164145835-e1dad409-04ad-41f7-9d08-87490676ae08.png)

Reply:
Thank you. I fixed it in #844.